### ISSUE TITLE
Unhandled Exception: Pointer<Uint32> has no instance getter value

### ISSUE THREAD

**Issue by timsneath** [dart-lang/sdk#44883]
DartCode fails to recognize the extension method `value` on `Pointer<Uint32>` in the debugger window, as shown below:
![image](https://user-images.githubusercontent.com/2319867/107159135-14f11000-6943-11eb-9402-b7c7fbaee684.png)

This method very much exists:
https://api.dart.dev/be/177834/dart-ffi/Uint32Pointer.html

I don't know whether this is a DartCode issue, a debugger / analyzer issue, or an FFI issue, so tagging a few folk here who might be able to help me diagnose: @dcharkes @DanTup 

## Versions
Dart SDK version: 2.12.0-259.8.beta (beta) (Tue Feb 2 15:59:29 2021 +0100) on "windows_x64"
DartCode extension 3.19.2 on VS Code 1.53.0

**Comment by mraleph** [comment]
Duplicate of #44801

**Comment by mraleph** [comment]
tldr: VM does not pass enough information back to front-end to resolve extension methods when evaluating expressions with debugging.

**Comment by timsneath** [comment]
Thanks! Consider me an upvoter for the feature required :)
